src/components/Friends/Friend/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect, memo } from 'react'
+import './styles.scss'
+import propTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+
+
+const Friend = props => {
+  const { channel, getMembers, unreadCount, getUserByUsername, getusername, getPosts, getLatestMessage } = props
+  
+  const [members, setMembers] = useState([])
+  const [otherUser, setOtherUser] = useState({})
+  const [user, setUser] = useState({})
+  const [posts, setPosts] = useState({})
+
+  const imageUri = user!==undefined ? `http://${process.env.REACT_APP_MATTERMOST_URL}/api/v4/users/${user.id}/image` : null
+  const message=getLatestMessage(posts)
+ 
+  useEffect(() => {
+    getMembers(channel.id).then(data => setMembers(data.data))
+  }, [])
+
+  useEffect(() => {
+    if(members) {
+      const user=getusername(members)
```

Comment:
user is already declared in the upper scope, better use some other constant name here :)